### ISSUE TITLE
Toolbar in Tree view not screen readable

### ISSUE THREAD

**Issue by sandy081** [Microsoft/vscode#8181]
- VSCode Version: Latest insiders
- OS Version: OSX

Steps to Reproduce:
1. Enable Screen reader
2. Open Search view and search for some results
3. Navigate to remove action on search results

Remove action is not read as a button

**Comment by joaomoreno** [comment]
@sandy081 The contents of tree rows are not the tree's responsibility, this needs to be addressed by the search viewlet itself.

**Comment by sandy081** [comment]
Closing as this might not be an accessibility bug otherwise the accessibility team should have found it.
